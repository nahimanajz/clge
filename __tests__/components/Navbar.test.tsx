import { render, screen } from "@testing-library/react";
import NavBar from "@/components/Navbar";

describe("Navbar tests", () => {
  const onFilter = () => {};
  
  it("Should match navbar snapshot", () => {
    const { container } = render(<NavBar onFilter={onFilter} />);
    expect(container).toMatchSnapshot();
  });
  it("Should find app title ", () => {
  
    render(<NavBar onFilter={onFilter} />);
    const titleTxt = screen.getByText("Hotel&Co")
   expect(titleTxt).toBeInTheDocument();
 });
});
