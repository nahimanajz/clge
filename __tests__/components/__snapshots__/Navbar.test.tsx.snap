// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar tests Should match navbar snapshot 1`] = `
<div>
  <div>
    <nav
      class="bg-white fixed z-10 w-full mt-0"
    >
      <div
        class="flex justify-between flex-wrap items-center p-4"
      >
        <div>
          <a
            class="text-primary text-2xl font-semibold"
            href="/"
          >
             
            Hotel&Co
          </a>
        </div>
        <div
          class="relative"
        >
          <input
            class="shadow appearance-none border outline-none focus:border-primary rounded w-full py-2 p-3 pr-10 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
            placeholder="Search by country"
            type="text"
          />
          <div
            class="absolute inset-y-0 right-0 pr-3 flex items-center pointer-events-none"
          >
            <svg
              class="text-primary"
              fill="none"
              height="1em"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="11"
                cy="11"
                r="8"
              />
              <line
                x1="21"
                x2="16.65"
                y1="21"
                y2="16.65"
              />
            </svg>
          </div>
        </div>
      </div>
      <div />
    </nav>
  </div>
</div>
`;
